src/pages/documentation/release-notes/index.md

```diff
@@ -4,6 +4,14 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 # Release notes
 
+## February 17, 2023
+
+### Android Analytics 2.0.1
+
+* Made extension compatible with Kotlin 1.4.
+* Analytics shared state is made available on boot-up if there are any previously stored identifiers.
+* Shared state is only updated when an identifier is set and not on get requests.
+
```

Comment:
Do you want to add the Analytics 2.0.0 release notes here as well?

Reply:
Not in this PR, it will be added with the other extensions in the general notes